### ISSUE TITLE
Security check crashing game

### ISSUE THREAD

**Issue by tenplus1** [minetest/minetest#4943]
Trying to run minetest with hopper mod enabled, this is what I get each time I enable a mod and click save button on today's 5231 build:

```
Loaded texture: /usr/share/minetest/textures/base/pack/menu_bg.png
Loaded texture: /usr/share/minetest/games/minetest_game/menu/header.png
2016-12-21 09:41:20: ERROR[Main]: MAINMENU ERROR: Runtime error from mod '' in callback handleMainMenuButtons(): Settings: writing /home/john/.minetest/worlds/v/world.mt not allowed with mod security on.
2016-12-21 09:41:20: ERROR[Main]: stack traceback:
2016-12-21 09:41:20: ERROR[Main]: 	[C]: in function 'write'
2016-12-21 09:41:20: ERROR[Main]: 	/usr/share/minetest/builtin/mainmenu/dlg_config_world.lua:231: in function 'handle_buttons'
2016-12-21 09:41:20: ERROR[Main]: 	/usr/share/minetest/builtin/fstk/ui.lua:148: in function 'handle_buttons'
2016-12-21 09:41:20: ERROR[Main]: 	/usr/share/minetest/builtin/fstk/ui.lua:192: in function </usr/share/minetest/builtin/fstk/ui.lua:178>
Loaded texture: /usr/share/minetest/textures/base/pack/menu_bg.png
Loaded texture: /usr/share/minetest/games/minetest_game/menu/header.png
Loaded texture: /usr/share/minetest/games/minetest_game/menu/icon.png
Loaded texture: /usr/share/minetest/games/minimal/menu/icon.png

**Comment by sfan5** [comment]
first broken in 59f84ca0a07e50dd5ce050d38ae1aeb529bd25ac

**Comment by est31** [comment]
cc @ShadowNinja

**Comment by est31** [comment]
@ShadowNinja please note that even if this issue got fixed before the release (which is not advisable as it might break something), mod security still should not be re-enabled by default because of the general distrust of issues like this that may pop up.

As I've said in #4944, after 0.4.15 is released and this issue is fixed, re-enabling mod security is fine with me.

**Comment by paramat** [comment]
@tenplus1 this is hopefully fixed but please test ASAP and let us know in IRC, thanks.

**Comment by ShadowNinja** [comment]
The bug was fixed by cc7c31a5bc44ab0010997e86c0a8c9f5f832b398, it only occurred with mod security off.
